setup.py

```diff
@@ -665,7 +665,7 @@ def fortran_extensionlists(self):
 NAME = "Assimulo"
 AUTHOR = u"C. Winther (Andersson), C. Führer, J. Åkesson, M. Gäfvert"
 AUTHOR_EMAIL = "christian.winther@modelon.com"
-VERSION = "trunk" if version_number_arg == "Default" else version_number_arg
+VERSION = "3.5.0-dev" if version_number_arg == "Default" else version_number_arg
```

Comment:
I saw @jschueller fixed this in https://github.com/modelon-community/Assimulo/pull/94
@PeterMeisrimelModelon, are you OK if we close this?

Reply:
sure, closing